react/src/components/helpers/formhelpers/Radios.tsx

```diff
@@ -29,24 +30,34 @@ function Radios(props: RadiosProps) {
         {errors[name] ? <span className="govuk-error-message">{errors[name]?.message}</span> : null}
         
         <div className="govuk-radios">
-          {items.map((item) => {
+          {items.map((item, index) => {
             return (
-              <div className="govuk-radios__item" key={`${name}-${item}`}>
-                <input 
-                  className="govuk-radios__input" 
-                  id={`${name}-${item}`}
-                  name={name}
-                  type="radio" 
-                  value={item}
-                  checked={selected === item}
-                  onChange={e => {
-                    handleChange(e.target.value)
-                  }}
-                />
-                <label className="govuk-label govuk-radios__label" htmlFor={`${name}-${item}`}>
-                  {item}
-                </label>
-              </div>
+              <>
```

Comment:
need to add a key to this fragment as it's in a map e.g. `<React.Fragment key={`${name}-${item}`}>`